docs/index.yml

```diff
@@ -0,0 +1,193 @@
+swagger: "2.0"
+info:
+  version: 1.0.0
+  title: YnvestTubeApi
+host: "127.0.0.1:8000"
+consumes:
+  - "application/json"
+produces:
+  - "application/json"
+paths:
+  /users:
+    get:
+      summary: 'Get all users'
+      tags:
+        - user
+      responses:
+        '200':
+          description: 'Users list retrieved successfully'
+          schema:
+            $ref: '#/definitions/UserList'
+        '403':
+          description: Bad user id
+  /users/register:
+    get:
+      summary: "Register new user"
+      tags:
+        - user
+      responses:
+        "200":
+          description: 'User id for an app to use in future calls'
+          schema:
+            $ref: '#/definitions/UserId'
+  /users/{userId}:
+    get:
+      summary: "Get user's overall info cash, account creation date."
+      tags:
+        - user
+      parameters:
+        - in: path
+          name: "UserId"
+          required: true
+          schema:
+            $ref: '#/definitions/UserId'
+      responses:
+        '200':
+          description: 'User data'
+          schema:
+            $ref: '#/definitions/UserDetails'
+        '403':
+          description: Bad user id
+  /users/{UserId}/details:
```

Comment:
I would merge this with be upper enpoint to return all the data for user with a single request. We would be showing both information on the same view so it is pretty senseless to make 2 request sequntially when we can make one to get all the data we need at the moment